oic.r.cred.raml

```diff
@@ -82,8 +81,7 @@ traits:
                     "subjectuuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
```

Comment:
Since this is not an issue introduced by this change I would prefer not to address it in this one.

But I think it's actually okay, because on UPDATE, the credId will be discarded and reassigned by the device in order to ensure uniqueness.